pkg/rancher-desktop/preload/extensions.ts

```diff
@@ -595,6 +642,71 @@ class Client implements v1.DockerDesktopClient {
         };
       });
     },
+    subscribeToEvents: (options: DockerEventSubscriptionOptions = {}, callback: DockerEventCallback): DockerEventSubscription => {
+      const eventArgs = ['events', '--format', '{{json .}}'];
+
+      if (options.filters) {
+        if (options.filters.type) {
+          options.filters.type.forEach(type => {
```

Comment:
You don't need the `if`s if you use optional chaining:
```suggestion
      options.filters?.type?.forEach(type => {
```